nativescript-angular/router/ns-location-strategy.ts

```diff
@@ -6,6 +6,39 @@ import { NavigationTransition, Frame } from "tns-core-modules/ui/frame";
 import { isPresent } from "../lang-facade";
 import { FrameService } from "../platform-providers";
 
+export class Outlet {
+    showingModal: boolean;
+    modalNavigationDepth: number;
+    parent: Outlet;
+    isPageNavigationBack: boolean;
+    outletKeys: Array<string>;
```

Comment:
[Reminder] As discussed it would be useful to add a comment why Outlet can have multiple (two) keys.